decorate_test.go

```diff
@@ -34,56 +34,95 @@ func TestDecorateSuccess(t *testing.T) {
 	t.Run("simple decorate without names or groups", func(t *testing.T) {
 		t.Parallel()
 		type A struct {
-			name string
+			Name string
```

Comment:
curious why this was needed?

Reply:
It wasn't - I think a change I was playing around with locally got merged in with this branch. I undid these changes.